packages/terra-clinical-result/src/terra-dev-site/test/clinical-result/clinical-result/TruncatedResult.test.jsx

```diff
@@ -11,6 +11,18 @@ export default () => (
       }}
       isTruncated
     />
+    <ClinicalResult
+      eventId="111"
+      result={{
+        value: '354.23377777758678567867856785687678567856767868678567856785678567856785678567857',
+        unit: 'mmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHgmmHg',
+      }}
+      isTruncated
+      isModified
+      hasComment
+      conceptDisplay={Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral'
```

Comment:
I imagine this should be failing lint? This is what is breaking the heroku deployment.
```suggestion
      conceptDisplay="Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral Temperature Oral"
```